src/tools/interop/idt/requirements.txt

```diff
@@ -0,0 +1,3 @@
+zeroconf==0.74.0
```

Comment:
Is this package stable enough to depend on? It looks like they are up to 0.112.0 as of this morning. 0.74.0 was only from August 3. The package is under very active development?

Reply:
Many recent commits seem to be optimizations. From my perspective, it seems maintained. An alternative would be to use things in the SDK (e.g. minimal mDNS), but this requires building etc. Are there any alternatives that are lightweight?